src/extension/content-script/tasks.ts

```diff
@@ -67,6 +67,52 @@ export default function tasks(...args: Parameters<typeof realTasks>) {
     // for debug purpose
     return _tasks
 }
+
+const uriCanDoTask = (tabUri?: string, targetUri?: any) => {
+    if (tabUri?.startsWith(targetUri)) return true
+    return false
+}
+
+export function wrappedTasks(...args: Parameters<typeof realTasks>) {
```

Comment:
Any document for these wrapped tasks?

Reply:
I think these changes should not appear here...